webapp/app/templates/components.html

```diff
@@ -35,28 +35,32 @@
     {% endif %}
 {%- endmacro %}
 
-{% macro checkbox(field, classes="") %}
+{% macro checkbox(field, classes="", first_field=False) %}
     <div class="form-row checkbox">
-        {{ _field(field) }}
+        {{ _field(field, first_field=first_field) }}
         <label class="checkmark" for="{{ field.name }}"></label>
         {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0") }}
     </div>
 {% endmacro %}
 
-{% macro consent_box(field, classes="") %}
+{% macro consent_box(field, classes="", first_field=False) %}
     <div class="form-row checkbox consent-box col-lg-10">
-        {{ _field(field) }}
+        {{ _field(field, first_field=first_field) }}
         <label class="checkmark" for="{{ field.name }}"></label>
         {{ field_label(field, class="col-sm-10 col-form-label ml-3 pt-0") }}
     </div>
 {% endmacro %}
 
-{% macro radio_field(field) -%}
+{% macro radio_field(field, first_field=False) -%}
     <fieldset>
         {{ field_label(field) }}
         {% for subfield in field %}
             <div>
-                {{ subfield(class="") }}
+                {% if (first_field or field.errors) and loop.index0==0 %}
```

Comment:
Woooow! What that? I've never seen that! I'd say *interesting* syntax. :D But I like it^^

Reply:
You mean loop.index0? I've looked it up and Jinja has a lot of information about the loop stored in the loop variable (s. [here](https://jinja.palletsprojects.com/en/2.11.x/templates/#for))